model-optimizer/extensions/back/ConvolutionNormalizer.py

```diff
@@ -24,18 +24,18 @@ def resolve_convolution_with_group(node: Node, group: int, ir_version: str):
     assert len(weights_shape) in [3, 4, 5]
     assert weights_shape[0] % group == 0
 
-    assert int64_array(node.output).ndim == 0
     if ir_version == 'V7':
         if weights_shape[0] == node.output:
             # weights are already is in [G*O I X Y] format
             return
-        new_shape = int64_array([node.output, -1, *weights_shape[2:]])
-
+        new_shape = shape_array([node.output, -1, *weights_shape[2:]])
```

Comment:
possibly dynamic_dimension_value (-1000007) should be used instead of -1

Reply:
No, here the `-1` means "all other elements" and is not related to dynamism.